examples/step_operator_remote_training/README.md

```diff
@@ -0,0 +1,186 @@
+# Train models on remote environments
+
+This example shows how you can use the `StepOperator` class to run your training jobs on remote backends.
+
+The step operator defers the execution of individual steps in a pipeline to specialized runtime environments that are optimized for Machine Learning workloads.
+
+## Overview
+Here we train a simple sklearn classifier on the MNIST dataset using one of three step operators:
+
+- AWS Sagemaker
+- GCP Vertex AI
+- Microsoft AzureML
+
+Currently, step operators only work with a local orchestrator but support for cloud orchestrators is on the way soon!
+
+## Installation
+In order to run this example, you need to install and initialize ZenML and the necessary integrations:
+
+```shell
+# install CLI
+pip install zenml
+
+# install ZenML integrations
+zenml integration install aws s3 sagemaker sklearn vertex gcp azure azureml
```

Comment:
Should we split this up a bit so its easier to only copy paste the relevant one of the step operators?